src/common/SPELLS_WARLOCK.js

```diff
@@ -703,4 +703,11 @@ export default {
     name: 'Dreaded Haste',
     icon: 'spell_shadow_metamorphosis',
   },
+
+  // Demonology legendary effects
+  WAKENERS_LOYALTY_BUFF: {
```

Comment:
Use the spell's actual name when possible. This should just be WAKENERS_LOYALTY. If a spell's name makes it unclear what it's from (like a buff with a different name from the legendary that procced it), you can add clarification in a comment. In this case, I don't think a comment is needed.

Reply:
I initially thought of it to be clear distinction between the item and the buff but considering one is in ITEMS and second in SPELLS, that kinda contradicts my argument.

Reply:
Yeah there are a lot of cases with same name in ITEMS and SPELLS